.gitattributes

```diff
@@ -11,4 +11,5 @@
 *.yml text eol=lf
 *.csv text eol=lf
 *.hl7 text eol=lf
+.environment/sftp-conf/ssh_host_*_key text eol=lf

```

Comment:
This fixes an issue reported by @mauricereeves on these files being converted by Git on Windows to CRLF